From bf43d12436e99c995ecb7ebe94b2c6a1070810c1 Mon Sep 17 00:00:00 2001
Message-Id: <bf43d12436e99c995ecb7ebe94b2c6a1070810c1.1342518105.git.minovotn@redhat.com>
In-Reply-To: <27a73856ecc481c66c7afac8171f753887f32e31.1342518105.git.minovotn@redhat.com>
References: <27a73856ecc481c66c7afac8171f753887f32e31.1342518105.git.minovotn@redhat.com>
From: Luiz Capitulino <lcapitulino@redhat.com>
Date: Tue, 5 Jun 2012 14:58:18 +0200
Subject: [PATCH 09/41] qemu-ga: Fix spelling in documentation

RH-Author: Luiz Capitulino <lcapitulino@redhat.com>
Message-id: <1338908331-15633-4-git-send-email-lcapitulino@redhat.com>
Patchwork-id: 39900
O-Subject: [PATCH RHEL6.4 qemu-kvm 03/36] qemu-ga: Fix spelling in documentation
Bugzilla: 827612
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>

Partial backport of upstream commit e7d81004e486b0e80a674d164d8aec0e83fa812f.

Signed-off-by: Luiz Capitulino <lcapitulino@redhat.com>
---
 qapi-schema-guest.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qapi-schema-guest.json |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qapi-schema-guest.json b/qapi-schema-guest.json
index 59bb538..6a75552 100644
--- a/qapi-schema-guest.json
+++ b/qapi-schema-guest.json
@@ -13,7 +13,7 @@
 # partially-delivered JSON text in such a way that this response
 # can be obtained.
 #
-# Such clients should also preceed this command
+# Such clients should also precede this command
 # with a 0xFF byte to make such the guest agent flushes any
 # partially read JSON data from a previous session.
 #
-- 
1.7.10.4

